tests/integration_tests/test_integration_mover.py

```diff
@@ -0,0 +1,238 @@
+                                 'skip_mover': True}
+                delivery_resp = yield self.http_client.fetch(self.get_url(delivery_url), method='POST', body=json.dumps(delivery_body))
+                delivery_resp_as_json = json.loads(delivery_resp.body)
+                delivery_link = delivery_resp_as_json['delivery_order_link']
+
+                status_response = yield self.http_client.fetch(delivery_link)
+                self.assertEqual(json.loads(status_response.body)["status"], DeliveryStatus.delivery_skipped.name)
+
+    def test_cannot_stage_the_same_project_twice(self):
+        # Note that this is a test which skips mover (since to_outbox is not expected to be installed on the system
+        # where this runs)
+
+        with tempfile.TemporaryDirectory(dir='./tests/resources/projects') as tmp_dir:
+
+            # Stage once should work
+            dir_name = os.path.basename(tmp_dir)
+            url = "/".join([self.API_BASE, "stage", "project", dir_name])
+            response = self.fetch(url, method='POST', body='')
+            self.assertEqual(response.code, 202)
+
+            # The second time should not
+            response = self.fetch(url, method='POST', body='')
+            self.assertEqual(response.code, 403)
+
+            # Unless you force the delivery
+            response = self.fetch(url, method='POST', body=json.dumps({"force_delivery": True}))
+            self.assertEqual(response.code, 202)
+
+    @gen_test
+    def test_can_stage_and_deliver_clean_flowcells(self):
+        with tempfile.TemporaryDirectory(dir='./tests/resources/runfolders/',
+                                         prefix='160930_ST-E00216_0555_BH37CWALXX_') as tmpdir1,\
+             tempfile.TemporaryDirectory(dir='./tests/resources/runfolders/',
+                                         prefix='160930_ST-E00216_0556_BH37CWALXX_') as tmpdir2:
+                self._create_projects_dir_with_random_data(tmpdir1, 'XYZ_123')
+                self._create_projects_dir_with_random_data(tmpdir2, 'XYZ_123')
+
+                url = "/".join([self.API_BASE, "stage", "project", 'runfolders', 'XYZ_123'])
+                payload = {'delivery_mode': 'CLEAN'}
+                response = yield self.http_client.fetch(self.get_url(url), method='POST', body=json.dumps(payload))
+                self.assertEqual(response.code, 202)
+
+                payload = {'delivery_mode': 'CLEAN'}
+                response_failed = yield self.http_client.fetch(self.get_url(url), method='POST', body=json.dumps(payload), raise_error=False)
+                self.assertEqual(response_failed.code, 403)
+
+                response_json = json.loads(response.body)
+
+                staging_status_links = response_json.get("staging_order_links")
+
+                for project, link in staging_status_links.items():
+                    self.assertEqual(project, 'XYZ_123')
+
+                status_response = yield self.http_client.fetch(link)
```

Comment:
Same for loop issues here as in the DDS integration tests (if i'm not mistaken ofc).